### DISCUSSION TITLE
Why are NDH alerts do not display the source IP directly in the alerts dashboard?

### DISCUSSION THREAD

**Discussion by b-u-g-s** [Security-Onion-Solutions/securityonion#8690] [General] [answered]
Hi,

When accessing the fake Synology web portal of my NDH I can see the alert in the ALERTS dashboard of SO
I can see the source IP in the event_data.message when I drill down but why is this not populated in the alert itself in the "source.ip" column ? 
In fact the only columns that are populated are: 
timestamp
rulename
severity
uuid 
and category

we can still get the information by looking at the event_data.message but shouldn't all the columns be populated?
It could be useful for a quick view and sorting

Cheers,
B.

**Comment by weslambert** [comment, marked as answer]
You can toggle the display view in `Alerts`, like so:

![image](https://user-images.githubusercontent.com/16829864/189347641-4be03533-2fdf-4799-95e0-58c96a8bd452.png)

You can also temporarily enable advanced features from the `Options` menu for more granular control.

**Reply by b-u-g-s** [reply]
Thanks a lot Wes!
